src/client.py

```diff
@@ -39,6 +39,13 @@ def start(self):
                 self.state.game_starts.acquire()
                 logger.debug("Main rendere proceed to the next scnee")
 
+            if not scene:
```

Comment:
i belive this actually won't work -> as long as the scene ends, pygame will quit (through an EOFError), but would like to figure out why

Reply:
Should we merge this then?